### PR TITLE
Build Tools by Ivan Genchev

### PR DESCRIPTION
**Pull request by freekman**
1)Maven task

### CODE REVIEW

File: Business/src/test/java/com/clouway/AppTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.clouway;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest
```
Comment (mgenov) [inline Business/src/test/java/com/clouway/AppTest.java]: Please, use junit 4

File: Persistence/pom.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
+    </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```
Comment (mgenov) [inline Persistence/pom.xml]: junit 4.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          </descriptorRefs>
+        </configuration>
+      </plugin>
+
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
```
Comment (mgenov) [inline pom.xml]: looks like multiple junit definitions

File: Business/src/test/java/com/clouway/AppTest.java (hunk 1 above)
Comment (freekman) [inline Business/src/test/java/com/clouway/AppTest.java]: Done

File: Persistence/pom.xml (hunk 2 above)
Comment (freekman) [inline Persistence/pom.xml]: Done

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**